inc/classes/subscriber/Media/class-webp-subscriber.php

```diff
@@ -266,11 +266,9 @@ public function webp_section_description( $cache_webp_field ) {
 			// 5, 8.
 			$cache_webp_field['helper'] = sprintf(
 				// Translators: %1$s = plugin name(s), %2$s = opening <a> tag, %3$s = closing </a> tag.
-				esc_html( _n( 'You are using %1$s to serve WebP images so you do not need to enable this option. If you prefer to have WP Rocket serve WebP for you instead, please disable them from serving in %1$s. %2$sMore info%3$s', 'You are using %1$s to serve WebP images so you do not need to enable this option. If you prefer to have WP Rocket serve WebP for you instead, please disable them from serving in %1$s. %2$sMore info%3$s', count( $serving ), 'rocket' ) ),
-				esc_html( wp_sprintf_l( '%l', $serving ) ),
-				'<a href="' . esc_url( $webp_beacon['url'] ) . '" data-beacon-article="' . esc_attr( $webp_beacon['id'] ) . '" target="_blank" rel="noopener noreferrer">',
-				'</a>'
-			);
+				esc_html( _n( 'You are using %1$s to serve WebP images so you do not need to enable this option. If you prefer to have WP Rocket serve WebP for you instead, please disable them from serving in %1$s.', 'You are using %1$s to serve WebP images so you do not need to enable this option. If you prefer to have WP Rocket serve WebP for you instead, please disable them from serving in %1$s.', count( $serving ), 'rocket' ) ),
+				esc_html( wp_sprintf_l( '%l', $serving ) )
+				);
```

Comment:
What's the reason to remove the beacon link on all those helper?

Reply:
i added The Beacon link

this class-webp-subscriber.php is adding a warning block  to the "webp checkox field" but the  "one-click addon" field does not have a warning block in it 
but we display the warning message in the "one-click addon"  field helper

should I add the warning block in "one-click addon"  field and edit the js to make "one-click addon" `wpr-radio-ui` work with the warning message button?

Reply:
I agree with you we can adapt the code for what exists in the one-click addon section